src/main/java/com/example/App.java

```diff
@@ -5,6 +5,13 @@
 public class App {
     public App() throws IOException {
 //        super();
+        // add more comments
+        if (true) {
+                // and more bad code
+        }
+        if (false) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://docker.for.mac.localhost:9000/coding_rules#rule_key=squid%3AS2583)